.env.example

```diff
@@ -47,3 +47,9 @@ SC_MONGODB_URI=""
 # When enabled, deployments will log to both JSON files AND MongoDB
 # When disabled, deployments work exactly as before (JSON files only)
 ENABLE_MONGODB_LOGGING=false
+
+# Safe Transaction Signing Configuration
+# Enable Safe transaction hash signing (set to true to enable, false to disable)
+# When enabled, uses eth_sign to sign the Safe tx hash instead of EIP-712 typed data
+# This is useful for hardware wallets (e.g. Ledger) that may reject large EIP-712 payloads
+ENABLE_SAFE_TX_HASH_SIGNING=false
```

Comment:
This works fine just wondering if it would be better to just enabled it as a fallback by default when that particular error occurs and then prompt to try Safe TX Hash signing. E.g. "There was an error signing this TX would you like to try Safe TX Sigining?"

I just fear the more new env vars there are in this file the more errors can happen if people haven't updated their config etc.